sos/plugins/kernel.py

```diff
@@ -133,6 +136,12 @@ def setup(self):
             clocksource_path + "current_clocksource"
         ])
 
+        if self.get_option('all_logs'):
+            self.trace_timeout=0
+
+        self.add_cmd_output("cat /sys/kernel/debug/tracing/trace",
```

Comment:
Nack on calling cat to collect this. Also, the whole-plugin timeout should be killing this collection well before the 2-hour mark unless you're running sosreport with `--plugin-timeout=0` or similar.

Personally I think we should just add this path as a forbidden path wholesale as I don't think we've ever purposely intended to capture this path. However, if it's useful it should be gated to either add the path as forbidden or not, not add it and then circumvent the mechanism we just leveraged.

If this is actually useful, we could gate it behind a plugin option, e.g. `-k kernel.trace=on`. This should be 'off' by default which should trigger adding it as a forbidden path. If the option is set to 'on' then we maintain the behavior today.

Reply:
Thank you for your comment.
I'll consider how to switch the collection of path using new plugin option.
I close this PR and create a new one.